packages/vats/src/core/types.js

```diff
@@ -238,12 +244,12 @@
  *   runBehaviors: (manifest: unknown) => Promise<unknown>,
  *   modules: Record<string, Record<string, any>>,
  * }} BootstrapPowers
- * @typedef { WellKnownSpaces & ChainBootstrapSpace & PromiseSpaceOf<{
+ * @typedef { WellKnownSpaces & PromiseSpaceOf<ChainBootstrapSpaceT & {
  *     vatAdminSvc: VatAdminSvc,
- *   }> & { produce: {
- *     loadVat: Producer<VatLoader<unknown>>,
- *     loadCriticalVat: Producer<VatLoader<unknown>>,
- *   }}
+ *   }, {}, {
```

Comment:
Interesting... I was afraid these types were bordering on unmaintainable, but you seem to not just work with them but lean into them. Nice to have somebody else figure out what's going on in these parts :)